src/style.css

```
@@ -0,0 +1,99 @@
+@font-face {
+  font-family: Avenir;
+  font-weight: 300;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+body {
+  font-family: Avenir, Arial, Helvetica, sans-serif;
+  font-weight: 300;
+}
+
+form {
+  margin: 30px 0;
+}
+
+* {
```

```suggestion
html {
```
better to apply font-family to `body` or `html`